pkg/apis/applicationnetworking/v1alpha1/accesslogpolicy_types.go

```diff
@@ -0,0 +1,65 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/gateway-api/apis/v1alpha2"
+
+	"github.com/aws/aws-application-networking-k8s/pkg/k8s"
+	"github.com/aws/aws-application-networking-k8s/pkg/model/core"
+)
+
+const (
+	AccessLogPolicyKind = "AccessLogPolicy"
+)
+
+// +genclient
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:categories=gateway-api,shortName=tgp
```

Comment:
`shortName=alp`

Reply:
Fixed next commit, thanks for catching this!